charts/buffer/templates/ingress/ingress.yaml

```diff
@@ -1,23 +1 @@
-{{ if .Values.ingress.enabled -}}
```

Comment:
Are there any contexts where ingress is disabled? Making a library chart respect an enabled flag is a bit weird, but it is doable I think. Here's how I got it working if you think it is worth it:
- Keep the [if statements](https://github.com/broadinstitute/terra-helm/blob/master/charts/leonardo/templates/migration.yaml#L1) around where you render the template out into the main chart
- Also include the [if statements](https://github.com/broadinstitute/terra-helm/blob/master/charts/liquibase-migration/templates/_job.yaml#L2) in the library chart itself for good measure
- Make the entire dependency [conditional](https://github.com/broadinstitute/terra-helm/blob/master/charts/leonardo/Chart.yaml#L16) on it being enabled